dozer/cogs/roles.py

```diff
@@ -488,6 +506,12 @@ async def give(self, ctx, member: discord.Member, *, role: discord.Role):
     `{prefix}give cooldude#1234 Java` - gives cooldude any role, giveable or not, named Java
     """
 
+    @cog_ext.cog_slash(name="give", description="Gives role(s) to given members.")
```

Comment:
I don't think commands that can't be used by all members should show up in the slash list because it will show as an available option even though it isn't